data/family/RedHat-8.yaml

```diff
@@ -0,0 +1,3 @@
+---
+rabbitmq::python_package: 'python3'
+rabbitmq::repos_ensure: true
```

Comment:
I see why you're doing this here, but I think probably we shouldn't use non-default params for one specific OS version for this.

If you want to switch the default to repos_ensure as a major (breaking) change, I'm onboard with that, though it may be a bit of an involved process.